lib/facter/unbound_version.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+unbound_bin = '/usr/sbin/unbound'
```

Comment:
Hmm, my FreeBSD servers might fail with this path.

Reply:
are you sure my bsd server is a bit behind but

```
ssh% uname -a
FreeBSD ssh.example.org 11.0-RELEASE-p15 FreeBSD 11.0-RELEASE-p15 #0: Tue Nov 14 08:04:40 UTC 2017     root@amd64-builder.daemonology.net:/usr/obj/usr/src/sys/GENERIC  amd64
ssh% which unbound
/usr/sbin/unbound

Reply:
I'm not certin.  This is what I was thinking though: https://github.com/xaque208/puppet-unbound/blob/master/data/os/FreeBSD.yaml#L3  Since we install the package with this module, we should probably be using the package's unbound.  I assume the `/usr/sbin/unbound` is from the unbound in base, which I think lacks TLS support.  Perhaps `which -a unbound`?

Reply:
yes you could be right, ill set up some beaker tests to  validate all this.  obviouly the bsd can not be done by travis unless you know something i dont but ill create some tests eaither way

Reply:
Personally I've not used beaker much, but test on long lived installations.  I think some unit tests for the platform would be fine, but however you want to approach that is fine.  I  still need to get openbsd updated in the facterdb code, which I've been meaning to do for months now.